django/db/models/aggregates.py

```diff
@@ -107,17 +116,17 @@ class Count(Aggregate):
     name = 'Count'
     output_field = IntegerField()
     allow_distinct = True
+    empty_aggregate_value = 0
 
     def __init__(self, expression, filter=None, **extra):
         if expression == '*':
             expression = Star()
         if isinstance(expression, Star) and filter is not None:
             raise ValueError('Star cannot be used with filter. Please specify a field.')
+        if extra.get('default') is not None:
+            raise ValueError('Count does not support the default argument.')
```

Comment:
I wonder if we should move this to `Aggregate.__init__` instead and base it of `self.empty_aggregate_value`